application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -278,12 +278,13 @@
           #set ($foundUser = false)
           #foreach ($voteObj in $doc.getObjects('XPoll.XPollVoteClass'))
             #set ($user = $voteObj.user)
-            #set ($isCurrentUser = $user == $xcontext.user)
+            #set ($isCurrentUser = $user.equals($xcontext.user))
```

Comment:
Equals is not required here because both are strings. Now, it's good to know that this works because:

* ``$xcontext.user`` is using the compact wiki entity reference serializer
* and the poll document is the current document (so the current user reference is serialized relative to the poll document reference)

Otherwise, if that were not true then you would have to resolve the voter reference yourself and compare with the current user reference.

```suggestion
            #set ($voterReference = $services.model.resolveDocument($voteObj.user))
            #set ($isCurrentUser = $voterReference.equals($xcontext.userReference))
```